### DISCUSSION TITLE
Support hex colors with underscore ( _ )

### DISCUSSION THREAD

**Discussion by darkomtc11** [mallowigi/color-highlighter#176] [Ideas]
Example: `0xFF_FF_FF` 
It's in typescript, and when i remove the underscores, your plugin works fine.

**Comment by mallowigi** [comment]
It's not a common format, where do you see it in use?

**Reply by darkomtc11** [reply]
Underscores are a part of a number. They are used for better readability, for example `1_000_000`. In our project Lint forces hex colors to be with underscores, and the plugin doesn't recognize it.

**Reply by mallowigi** [reply]
Sorry, that's not a valid reason for me. I've never encountered such a rule and it doesn't justify development for a single use case. You're welcome to make a PR if you want though.
